src/pages/DetailsPage.js

```diff
@@ -65,129 +66,138 @@ const getPhoneNumber = (details) => {
     return Str.removeSMSDomain(details.login);
 };
 
-const DetailsPage = (props) => {
-    const details = props.personalDetails[props.route.params.login];
-    if (!details) {
-        // Personal details have not loaded yet
-        return <FullscreenLoadingIndicator />;
+class DetailsPage extends React.PureComponent {
+    componentDidMount() {
+        if (lodashGet(this.props.route.params, 'login')) {
+            return;
+        }
+        Navigation.dismissModal();
```

Comment:
May be can add a log or comment here, what do you say @iwiznia?

Reply:
Adding a comment sounds good